src/store/modules/search.store.js

```diff
@@ -152,6 +153,44 @@ const actions = {
                     coordinates: entry.coordinate,
                     dispatcher: dipsatcherSelectResultEntry,
                 })
+                break
+            case SearchResultTypes.FEATURE:
+                if (entry.extent.length === 2) {
+                    dispatch('zoomToExtent', { extent: entry.extent, dispatcher })
+                } else if (entry.zoom) {
+                    dispatch('setCenter', {
+                        center: entry.coordinate,
+                        dispatcher: dipsatcherSelectResultEntry,
+                    })
+                    dispatch('setZoom', {
+                        zoom: entry.zoom,
+                        dispatcher: dipsatcherSelectResultEntry,
+                    })
+                }
+
+                // Automatically select the feature
+                try {
+                    getFeature(
+                        getters.getActiveLayersById(entry.layerId)[0],
```

Comment:
might be a good idea to add the layer object to the search result instead. It might be available when the search request is fired in `search.api.js`, so adding it to the typedef `LayerSearchResult` is preferable